### ISSUE TITLE
Inconsistencies in scaling of toolbar buttons

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#2110]
**Describe the bug**

This is about the controls' behavior in response to text scale settings in the system Settings app.

![image](https://user-images.githubusercontent.com/13142665/76576194-9f549980-647e-11ea-82db-07ff2a9365fe.png)

SplitButton icon and AppBarButton icons don't seem to scale their icons while the InkToolbar icons do scale and so does the dropdown icon on the SplitButton.

The question is - are the CommandBar and SplitButton icons supposed to scale? Are the InkToolbar icons supposed to not scale?

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/25339200)

**Comment by Felix-Dev** [comment]
So I checked this repo's history (as there have been a number of scaling issues reported) and the referrenced [documentation](https://docs.microsoft.com/en-us/windows/uwp/design/input/text-scaling#dont-scale-font-based-icons-or-symbols) which says:
> When using font-based icons for emphasis or decoration, disable scaling on these characters.

To my understanding this is not the case when only using icons (without accompanying text) so I'd say icons should be scaled here then (as the customer is using the text scaling setting to better see the different UI elements and there is no describing text next to the icons to scale). 

In this case then, this should mean the CommandBar icons in the screenshot above should be scaled while the SplitButton icon should be not, as it's "decorative". For the dropdown icon we have issue https://github.com/microsoft/microsoft-ui-xaml/issues/1519 to look at and it seems the general consensus over there is not to scale the icon.

Ultimately, I would ask myself: The customer is scaling text to be bigger in the system because they presumably have issues identifying (textual) UI. If they have these issues, doesn't that also mean they might have a hard time identifying unscaled icons? And if those icons have no textual description next to them which could be scaled, how else is the customer supposed to better identify them? Using the Magnifier after already setting the text size to be bigger? Sounds like one step too many here for me....

**Comment by kmgallahan** [comment]
@YuliKl 

See #1519 and @jevansaks response [here](https://github.com/microsoft/microsoft-ui-xaml/issues/1650) for #1635:

> I think the default answer is: don't scale icons, even if they're glyph-based. The user wanted text to be bigger but if we made icons bigger too it would be the same behavior as DPI scaling for the most part. The spirit of the setting is that just text gets bigger.

**Comment by StephenLPeters** [comment]
Split button has a clear bug here, the down arrow should not be clipped. I'm not sure what the solution to the app bar buttons is, maybe fore PM to decide?

**Comment by kmgallahan** [comment]
@StephenLPeters 

The clipping should have been fixed in WinUI via #1525. I presume Snip & Sketch is using the platform controls though.

**Comment by ackrumb** [comment]
Nope, Snip and Sketch uses WinUI. My colleague Filip can say more, I'll ping him.

**Comment by xyzzer** [comment]
That would explain it. Snip & Sketch last updated WinUI 9/26, while #1525 was fixed 10/31. We'll look into updating it when we get a chance.
